src/components/RegionSelector/index.tsx

```diff
@@ -82,12 +87,15 @@ const RegionSelector: React.FC<RegionSelectorProps> = ({
                         }) ?? selectedRegion.english_name
                       : isUserSetting && selectedRegion?.iso_3166_1 !== 'all'
                       ? intl.formatMessage(messages.regionServerDefault, {
-                          applicationTitle: currentSettings.applicationTitle,
                           region: currentSettings.region
                             ? intl.formatDisplayName(currentSettings.region, {
                                 type: 'region',
                                 fallback: 'none',
-                              }) ?? currentSettings.region
+                              }) ??
+                              regions?.find(
+                                (region) =>
+                                  region.iso_3166_1 === currentSettings.region
+                              )?.english_name
```

Comment:
We can clean this up by finding the fallback name outside of the return instead of inlining this find.